content/faq/index.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Text-to-Speech on Android
+description: Guide on how to make TTS work on Android
+weight: 100
+extra:
+    menu_title: FAQ
+---
+
+TLDR:
+
+Organic Maps uses the system text-to-speech (TTS) engine for voice instructions. The default engines vary by device. The choices can include Google Text-to Speech, device manufacturer's engine or a third-party one. 
+
+You can use a third-party Text-to-speech engine which can be downloaded from Google Play or F-Droid. 
+
+[RHVoice](https://rhvoice.org/) is a free and open source speech engine which can be downloaded from [Google Play](https://play.google.com/store/apps/details?id=com.github.olga_yakovleva.rhvoice.android) and [F-Droid](https://f-droid.org/en/packages/com.github.olga_yakovleva.rhvoice.android/).
+
+## Instructions
+
+- Open the Settings app on your Android device.
+- Select Additional Settings and then select Accessibility.
+- Choose your preferred engine, speech rate and pitch.
+
+If you cannot find the relevant setting, open the settings app and search for Text-to-speech.
+
+P.S: Do note that these steps may vary based on the brand you are using.
+
+The option may not appear if you don't have a TTS installed on your device. Please install Google Speech Services from the [Play Store](https://play.google.com/store/apps/details?id=com.google.android.tts), which is used by many devices or you can install a FOSS third party one such as [RHVoice](https://play.google.com/store/apps/details?id=com.github.olga_yakovleva.rhvoice.android) from F-Droid.
+
+## Screenshots
+
+<table>
+    <tr>
+      <th></th>
+      <th></th>
+      <th></th>
+    </tr>
+    <tr>
+      <td><img src='/images/screenshots/tts_config_1.jpg' width='400' alt='Screenshot 1'></td>
```

Comment:
Simple tables are supported in markdown syntax, check templates/shortcodes/crypto_table.md